.github/workflows/build.yml

```diff
@@ -86,12 +85,8 @@ jobs:
           - oci-snapshotter-stargz
         typ:
           - integration
-          - dockerfile
```

Comment:
Is there a reason that the `./frontend/dockerfile` pkg is still referenced above? Do we have integration tests in there, I though we only had `dockerfile` tests there.

Reply:
Ah it's an oversight. Fixed.